sup3r/postprocessing/file_handling.py

```diff
@@ -27,7 +27,8 @@
 H5_ATTRS = {'windspeed': {'scale_factor': 100.0,
                           'units': 'm s-1',
                           'dtype': 'uint16',
-                          'chunks': (2000, 500)},
+                          'chunks': (2000, 500),
+                          'min': 0},
             'winddirection': {'scale_factor': 100.0,
```

Comment:
min/max on winddirection should be 0, 360 right?

Reply:
Do we want to do that though? 540 can be converted to 180 but if we force 540->360 that's no longer possible. There shouldn't be values outside of 0, 360 if the output handler is reached through the pipeline but if we use it directly to write arbitrary data we'd want to preserve the periodicity right?

Reply:
When you convert u/v to windspeed / dir you fix that ceiling/periodicity issue right? I think the output handler should always enforce physical limits. This is is especially important if you set your scale factor close to the bitrange. If you dont enforce a physical ceiling, you could have a bit overflow and end up with garbage.

Reply:
Yeah the conversion enforces that range but I was wondering about using the output handler without doing a pipeline run (which then wouldnt include that conversion). But that latter point makes sense.

Reply:
yeah i'd rather protect the physical range rather than allowing an unspecified range. Honestly the rex outputs handler should have protection against bit overflow. I could have sworn i wrote that somewhere at some point but i cant find it.